travis/build.sh

```diff
@@ -1,92 +0,0 @@
-#! /bin/bash
```

Comment:
This script should be restored, as it allows for quickly creating fully functional AppImages of linuxdeploy on a dev system. Even if it's no longer used, really. Eventually, we should just merge this again with the `build-in-docker.sh` script.